docs/user/FlowVariables.md

```diff
@@ -233,11 +229,18 @@ configuration file.
 | `ADDITIONAL_LIBS`        | Hardened macro library files listed here.                                                          |
 | `ADDITIONAL_GDS`         | Hardened macro GDS files listed here.                                                              |
 | `VERILOG_INCLUDE_DIRS`   | Specifies the include directories for the Verilog input files.                                     |
-| `CORNER`                 | PVT corner library selection.                                                                      |
+| `CORNER`                 | PVT corner library selection. Only available for ASAP7 and GF180 PDK.                              |
 | `DESIGN_NICKNAME`        | DESIGN_NICKNAME just changes the directory name that ORFS outputs to be DESIGN_NICKNAME instead of DESIGN_NAME in case DESIGN_NAME is unwieldy or conflicts with a difference design.                                                                                    |
 | `ABC_AREA`               | Strategies for Yosys ABC synthesis: Area/Speed. Default ABC_SPEED.                                 |
 | `PWR_NETS_VOLTAGES`      | Used for IR Drop calculation.                                                                      |
 | `GND_NETS_VOLTAGES`      | Used for IR Drop calculation.                                                                      |
+| `BLOCKS`                 | Blocks passed as blackbox into yosys. Do note that you have to specify block-specific inputs file in the directory mentioned by [Makefile](../main/flow/Makefile) |

```

Comment:
```suggestion
| `BLOCKS`                 | Blocks is a hardened macros through ORFS and integrate to top level design. Do note that you have to specify block-specific inputs file in the directory mentioned by [Makefile](../main/flow/Makefile). |

```

Reply:
@maliberty is description for `BLOCKS` is correct?

Reply:
@maliberty Any thoughts?

Reply:
Not quite. I would phrase it as this:
`Blocks used as hard macros in a hierarchical flow.`